release_notes.md

```diff
@@ -2,28 +2,21 @@
 
 ## Playbook Enhancements
 
-- Added a new playbook **Reset Scenario - Get Correlated Records** that fetches all the correlated records of the demo record created for the simulation
-
 - Following playbooks have been modified
 
   - **Run Selected Scenario** playbook modifications:
-
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
-
-    - A `Simulation` tag is added to the playbook
-
-    - The **Create Scenario** step is renamed to **Create Alerts** 
+    - A `SystemWaitForCompletion` tag is added to the playbook
```

Comment:
why added?

Reply:
This is to make the "Simulate Scenario" button in the Scenario record, to be disabled for the long-running scenario playbook. In current behaviour, the "Simulate Scenario" button remains active, leading to confusion at users and user may end up executing same scenario twice.